wisp-logging/build.gradle.kts

```diff
@@ -5,6 +5,7 @@ plugins {
 dependencies {
     api(libs.loggingApi)
     implementation(libs.slf4jApi)
+    implementation(project(":wisp-sampling"))
```

Comment:
I wasn't sure if this should be `implementation` or `api`.  Happy to take advice on this.